package.json

```diff
@@ -30,7 +30,10 @@
     "url": "https://github.com/jstransformers/jstransformer-lodash/blob/master/LICENSE.md"
   },
   "dependencies": {
-    "lodash": "^3.6.0"
+    "lodash.assign": "^3.2.0",
```

Comment:
I agree with @TimothyGu.  Deduping commands are within their rights to use any compatible version that satisfies all constraints.

In addition, it's useful to track which version the author actually tested against in case there are any accidental breaking changes and we end up having to track down a "last know good version"